src/bootstrap/download-ci-llvm-stamp

```diff
@@ -1,4 +1,4 @@
 Change this file to make users of the `download-ci-llvm` configuration download
 a new version of LLVM from CI, even if the LLVM submodule hasn’t changed.
 
-Last change is for: https://github.com/rust-lang/rust/pull/96971
```

Comment:
I don't think this applies here, `rustc_llvm` should be rebuilt for every user normally

Reply:
ok 👍🏻